src/solver/optimisation/opt_appel_solveur_lineaire.cpp

```diff
@@ -234,8 +237,17 @@ bool OPT_AppelDuSimplexe(PROBLEME_HEBDO* problemeHebdo,
     {
         solver = ORTOOLS_ConvertIfNeeded(&Probleme, solver);
     }
-    const std::string filename = createMPSfilename(optPeriodStringGenerator, optimizationNumber);
-    mpsWriterFactory mps_writer_factory(problemeHebdo->ExportMPS, problemeHebdo->exportMPSOnError, optimizationNumber, &Probleme, ortoolsUsed, solver);
+
+    auto adqPatchOptNumber
```

Comment:
Following the oral talk we just had, I think that the MPS filename should **not** be computed here, but at a higher in the call stack.
Here this name should be passed as an argument.
See this comment (not linked yet)